### PR TITLE
Updating indexing example with formatting changes

### PR DESCRIPTION
**Pull request by yzubkov**

### CODE REVIEW

File: examples/indexing.lua [hunk 1]
```diff
@@ -4,42 +4,45 @@ local col = db:getCollection("exampleCO")
...
-	unique=false, 
-	sparse=true, 
-	background=true, 
-	expireAfterSeconds=100, 
-	textIndexVersion=1,
-	weights={name=10, txt=3} 
-} );
-print('\nCreating text search index: '..tostring(res))
+local indx = col:createIndex({name = 'text', txt = 'text'}, { 
+--	name = 'myindex',
```
Comment (christopherjwang) [inline examples/indexing.lua]: why is this commented out?

File: examples/indexing.lua [hunk 2]
```diff
@@ -4,42 +4,45 @@ local col = db:getCollection("exampleCO")
...
+	default_language = 'english', 
+	language_override = 'lang', 
+	unique = false, 
+	sparse = true, 
+	background = true, 
+	expireAfterSeconds = 100, 
+	textIndexVersion = 1,
+	weights = {name = 10, txt = 3} 
+} )
+print('\nCreating text search index: '..indx)
```
Comment (christopherjwang) [inline examples/indexing.lua]: you want spaces before/after the `..`. So:

`print('\nCreating text search index: ' .. indx)`

File: examples/indexing.lua (hunk 1 above)
Comment (yzubkov) [inline examples/indexing.lua]: I think I was attempting to demonstrate that you can a valid index name returned even if you do not provide one as a parameter. I agree that this is not necessary, so I will uncomment this line.

File: examples/indexing.lua (hunk 2 above)
Comment (yzubkov) [inline examples/indexing.lua]: Got it! Thanks.

**Pull request closed without merging**